src/view/com/profile/ProfileFollowers.tsx

```diff
@@ -105,7 +109,7 @@ export function ProfileFollowers({name}: {name: string}) {
   return (
     <List
       data={followers}
-      renderItem={renderItem}
+      renderItem={info => renderItem(info, isMe)}
```

Comment:
Not sure if this is the right way to do it, if I need to `useCallback` here to prevent this from changing, or if this method is just "all wrong".